### PR TITLE
reverted the TOC and changelog to remove policy additions

### PR DESCRIPTION
**Pull request by CelesteDG**
Removed the policy entries in the changelog and reverted the TOC to remove the policy API topics (sources were kept)

### CODE REVIEW

File: api-reference/beta/toc.yml [hunk 1]
```diff
@@ -8912,67 +8912,25 @@
           items:
           - name: Get member objects
             href: api/orgcontact-getmemberobjects.md
-      - name: Policy
+      - name: Policies
```
Comment (dkershaw10) [inline api-reference/beta/toc.yml]: This should still say Policy, and the href should be to policy.md.

Comment (dkershaw10) [inline api-reference/beta/toc.yml]: This should still say Policy, and the href should be to policy.md.

File: api-reference/beta/toc.yml [hunk 2]
```diff
@@ -8915,64 +8915,22 @@
       - name: Policy
         href: resources/policy-overview.md
```
Comment (dkershaw10) [inline api-reference/beta/toc.yml]: No - resources/policy.md

The policy-overview.md is a new topic for the new API.

**Comment by CelesteDG** [comment]
Confirmed changes are rendering fine. Merging to master.

**Pull request merged**